### PR TITLE
Add readme to examples folder

### PR DESCRIPTION
**Pull request by TodePond**
We link to the examples folder in many places (eg: docs site, discord).

It's not immediately clear what to do, or where to look, when you get there.
Also, our examples are really helpful, but there's no top-level view of what they are, or what they're for.

This README gives you some pointers, so you can crack on with learning from the examples.

### Change Type

- [x] `documentation` — Changes to the documentation only[^2]

[^1]: publishes a `patch` release, for devDependencies use `internal`
[^2]: will not publish a new version

### Release Notes

- Documentation: Added a overview of all our examples in the examples folder readme.

### CODE REVIEW

**Comment by steveruizok** [comment]
I'd prefer to use a landing page on the examples site itself than have a separate list in a README.

File: apps/examples/README.md [hunk 1]
```diff
@@ -1 +1,39 @@
...
+| [`/error-boundary`](src/8-error-boundary/) | Using a custom error fallback for shapes. |
+| [`/hide-ui`](src/9-hide-ui/HideUiExample.tsx) | Using tldraw without any UI. |
+| [`/custom-components`](src/10-custom-components/CustomComponentsExample.tsx) | Using your own custom components for brushes, scribbles, and snaplines. |
+| [`/user-presence`](src/11-user-presence/UserPresenceExample.tsx) | Testing the user presence API. |
+| [`/ui-events`](src/12-ui-events/UiEventsExample.tsx) | Listening to UI events. |
+| [`/store-events`](src/13-store-events/StoreEventsExample.tsx) | Listening to store events. |
+| [`/persistence`](src/14-persistence/PersistenceExample.tsx) | Using local persistence. |
+| [`/zones`](src/15-custom-zones/ZonesExample.tsx) | Placing custom UI elements at the top of the default UI. |
+| [`/custom-styles`](src/16-custom-styles/) | Using custom style properties on a custom shape. |
+| [`/yjs`](src/yjs) | Live collaboration using Yjs. |
```
Comment (steveruizok) [inline apps/examples/README.md:39]:
```suggestion

```

**Comment by TodePond** [comment]
Removed the list, as we will be landing an examples landing page soon (https://github.com/tldraw/tldraw/pull/1688).

When that happens, we can change some of our links in the docs to the landing page instead, and link to the source from that.

**Comment by TodePond** [comment]
I will close this for now, and will look at updating our links to match the new examples landing page (in a different PR).

**Pull request closed without merging**